EmergencyStickerEncryption.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  EmergencyStickerEncryption.swift
+//  Krypt
+//
+//  Created by Sun Bin Kim on 21.06.19.
+//
+
+import CryptoSwift
+import Foundation
+
+public class EmergencyStickerEncryption {
+  /// Errors
+  ///
+  /// - invalidSalt: Failed to process salt
+  /// - failToGenerateKeyAndFingerprintFile: Failed to generate key and fingerprint file
+  public enum Error: LocalizedError {
```

Comment:
We don't nest `Error` types anymore afaic. Should move it out `EmergencyStickerEncryptionError: Error`